### PR TITLE
added the basics foir the dresses show view...

### PR DESCRIPTION
**Pull request by KeiranBrown1301**
Also I added a VERY VERY basic fixed footer in the shared folder.

### CODE REVIEW

File: app/views/dresses/show.html.erb [hunk 1]
```diff
@@ -1 +1,17 @@
-<%= @dress %>
```
Comment (CamillieFu) [inline app/views/dresses/show.html.erb:1]: Could you put everything inside one "container" class? :)

**Review by CamillieFu — CHANGES_REQUESTED** [review]
I think after putting everything in one "container" it looks good :)

**Comment by KeiranBrown1301** [comment]
Just added one big container around everything :)

**Review by CamillieFu — CHANGES_REQUESTED** [review]
just one ;)

**Comment by KeiranBrown1301** [comment]
Changed it again to remove the other containers.

**Pull request merged**